server.py

```
@@ -0,0 +1,273 @@
+            "response": [log_response],
+        },
+    )
+    return FastMCP.from_openapi(
+        openapi_spec=filtered_spec,
+        client=client,
+        name="X API MCP",
+    )
+
+
+def main() -> None:
+    host = os.getenv("MCP_HOST", "0.0.0.0")
```

Ruff (0.15.21)

[error] Possible binding to all interfaces

(S104)